components/webfield/ProgramChairConsole/Overview.js

```diff
@@ -954,7 +983,7 @@ const DescriptionTimelineOtherConfigRow = ({
                       scoresName
                     )}
                   >
-                    Reviewer Bids
+                    {prettyField(singularReviewerName)} Bids
```

Comment:
I'm not sure if we should show "Reviewers" everywhere instead of trying ti get the singular reviewer name

Reply:
Yeah I realized that we don't need to make _all_ the replacements. But for this specifically, this is what the overview tab looks like. I think it looks good, but I can change it back in some areas:

<img src="https://github.com/openreview/openreview-web/assets/22416602/af90453f-b09b-4a3c-936b-7bac82162d38" width="500">

Reply:
Looks good!

Right, for AAAI the Program Committee role is not updated to a singular version, that is why I think we should use `reviewers_name` everywhere.

Reply:
Ooh, so what about the default role names like reviewers, area chairs etc. that are plural by default?

Reply:
Showing "Reviewers Recruitment" is it too bad?

Reply:
I think it makes sense in some places to keep the singular name, but I can definitely remove it in other places.

Reply:
@melisabok I removed the use of singular names in some components, but I think it makes sense to keep it in the Overview tab, otherwise the headings read weirdly.

The only other place it's used is in the tabs, this is what it looks like:

<img width="500" alt="Screenshot of PC Console" src="https://github.com/openreview/openreview-web/assets/22416602/d9b7218f-c767-4ac6-a5ac-cabb189dcb2e">